db/migrate/20151103155048_add_btree_gist_extension.rb

```diff
@@ -4,13 +4,7 @@
 class AddBtreeGistExtension < ActiveRecord::Migration[4.2]
   def up
     if db_adapter_is? "PostgreSQL" then
-      create_extension :btree_gist
-    end
-  end
-
-  def down
-    if db_adapter_is? "PostgreSQL" then
-      drop_extension :btree_gist
+      enable_extension :btree_gist
```

Comment:
If we're editing existing migration files, it would be nicer to just delete this and `db/migrate/20151021082639_add_pg_trgm_support.rb`, and create a new migration for this, wouldn't it?

Reply:
How do you plan to create gist_index in migrations if this extension is enabled only in the end?

Reply:
I don't think this is neccesary at this time-point. If this is the only issue, then I believe this one can be merged.